scripts/mutect2_wdl/mutect2.wdl

```diff
@@ -71,15 +72,19 @@ workflow Mutect2 {
     File? gnomad_index
     File? variants_for_contamination
     File? variants_for_contamination_index
-    Boolean is_run_orientation_bias_filter
+    Boolean? is_run_orientation_bias_filter
```

Comment:
While we're at it, can we rename all of the "is_xxx" variables? I prefer `orientation_bias_filter` to `is_run_orientation_bias_filter`, for example. If you think of a better name I'll support that too, though it's difficult since we have to avoid the name collision with `run_orientation_bias_filter.` Don't worry about it if you disagree. It's just a pet peeve of mine...